CuratedContent/WorkingEffectivelyWithLegacyCode.md

```diff
@@ -42,13 +42,14 @@ The main contribution of this book is the "Legacy Code Change Algorithm":
 3. **Add new functionality with new tests** (usually following the Test Driven Development (TDD) process).
 4. **Refactor** tested code to remove duplication, clean up, etc.
 
-Above, Step 1 defines the targeted subset of legacy code that must have tests which depends on what needs to change, what can be sensed, and where dependencies can be broken.
-After the targeted legacy code has been covered with unit tests in Step 2, it should have close to 100% coverage (otherwise one can not claim that existing behavior is being preserved, unless the uncovered code and behavior is never used by anyone, in which case it should likely be removed).
-Any refactorings in Step 4 are safe to perform once the targeted code is sufficiently covered with tests.
+Step 1 defines the targeted subset of legacy code that must have tests which depend on what needs to change, what can be sensed, and where dependencies can be broken.
+After unit tests which cover the targeted code have been implemented in Step 2, it should have close to 100% coverage.
+Without full coverage, it is difficult to know if existing behavior is preserved.
+Any refactoring in Step 4 is safe to perform once the targeted code is sufficiently covered with tests.
 Refactoring is critical to improve the clarity and maintainability of the code.
-WARNING: The refactorings must not extend outside code that is sufficiently covered by tests!
+**WARNING:** Refactoring must not extend outside code that is sufficiently covered by tests!
```

Comment:
Again, it is also proper to say "The refactorings" but in this context it would also be okay to say "Refactors must not ..." or "Refactoring must not ...".